src/app/components/dialogs/baseDialogService.abstract.ts

```diff
@@ -54,21 +54,26 @@ export abstract class BaseDialogService {
         panelClass: 'base-dialog',
         data: dialogData,
         id: dialogId,
-        hasBackdrop: (null == parentElement),
-        disableClose: !closable,
+        hasBackdrop: null == parentElement,
+        disableClose: closable,
```

Comment:
@DanielLyleWilliamWarren  this was why cypress tests were failing about the share dialog. @Sveouu ive reverted this change to original so if this has unintended consequences (breaks other things!) let me know...